recipe/recipe.yaml

```
@@ -18,11 +18,11 @@ build:
 
 requirements:
   host:
-    - python
+    - python ${{ python_min }}
     - pip
     - setuptools
   run:
-    - python
+    - python >=${{ python_min }}
     - numba >=0.59.1
 
 tests:
```

Think we want something like this in the test section

```suggestion
tests:
  - requirements:
     run:
       - python ${{ python_min }}
       - pip
```

Basing this off [this example]( https://github.com/conda/ceps/blob/4f4ac7545d4ba42fb10f71e40ad66ec865daab2e/cep-0014.md?plain=1#L801-L803 )

---

Great. I wasn't sure how to handle tests in the new recipe format. Thanks for the suggestion.